packages/syft/src/syft/oblv/auth.py

```diff
@@ -0,0 +1,17 @@
+# stdlib
+from getpass import getpass
+from typing import Optional
+
+# third party
+from oblv import authenticate
+
+
+def login(apikey: Optional[str] = None):
+    if apikey is None:
+        apikey = getpass("Please provide your oblv apikey to login:")
+    try:
+        return authenticate(apikey)
+    except Exception as e:
+        print("Failed to Authenticate with message : ")
```

Comment:
Q: Can't we simply raise the exception?

Reply:
Solved by [3c9d137](https://github.com/OpenMined/PySyft/pull/7202/commits/3c9d1375f5cf244ccdde2ac9274a203c81463717)